.github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Add content to the website
+about: For adding content - can be completed entirely in CMS
```

Comment:
non-blocking suggestion

```suggestion
about: For adding content - can be completed entirely in CMS, or via a github pull request
```

You can update content with a pull request, yea?

Reply:
yes ✨